.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
There are two `.vscode` folders, one on the top level of the repo and one in the session material. do we need to keep both?

Reply:
I intentionally put both places, but the one in the root can be removed.